db/model/generator.js

```diff
@@ -446,22 +447,12 @@ module.exports = function generator(seq, dataTypes) {
    * @returns {Promise<Generator>}
    */
   Generator.prototype.assignToCollector = function () {
```

Comment:
Note that this doesn't return a promise anymore, or save to the db. I figured this would make more sense for being called from generator patch/put and collector start where it's already saving the generator later, so we're not saving to the db multiple times.

Reply:
this should be comment in the code